homeassistant/components/synology_srm/__init__.py

```diff
@@ -1 +1,73 @@
-"""The Synology SRM component."""
+"""The Synology SRM integration."""
+import asyncio
+
+import synology_srm
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .router import SynologySrmRouter
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
+PLATFORMS = ["device_tracker"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Synology SRM component."""
+    return True
```

Comment:
You can easily support import config from the `config.yaml`, why not ?

See #29927 or #32704

Reply:
@aerialls

Reply:
Following [the future of YAML](https://www.home-assistant.io/blog/2020/04/14/the-future-of-yaml/#we-are-still-an-open-source-project), I prefer to delete the YAML integration as soon as possible. I want to add in the near future some dynamic parameters to give the user some control over what is being tracked (and I'm not comfortable or have the time to maintain both system in parallel).